cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -0,0 +1,15 @@
+
+#include <gtest/gtest.h>
+#include <ray/api.h>
+#include <ray/api/ray_config.h>
+
+using namespace ray::api;
+
+TEST(RayClusterModeTest, PutTest) {
```

Comment:
Can we add a TaskTest here too?

Reply:
Is not easy to add test because I have not resolved link problem. The test code and default_worker should be one binary now. But I will try to add the test.